rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbLogNode.java

```diff
@@ -83,6 +85,17 @@ public void onFailure(Throwable t) {
         }, MoreExecutors.directExecutor()); //usually js responses runs on js callback executor
     }
 
+    boolean isStandard(TbLogNodeConfiguration conf) {
+        Objects.requireNonNull(conf, "node config is null");
+        final TbLogNodeConfiguration defaultConfig = new TbLogNodeConfiguration().defaultConfiguration();
+        switch (conf.getScriptLang()) {
+            case JS: return defaultConfig.getJsScript().equals(conf.getJsScript());
+            case TBEL: return defaultConfig.getTbelScript().equals(conf.getTbelScript());
+        }
+        log.warn("No rule to define isStandard script for script language [{}], assuming that is non-standard", conf.getScriptLang());
```

Comment:
This is not possible, but usually, we use the "default" keyword in the "switch" statement to implement such behavior.

Reply:
Done as you suggested.
![image](https://user-images.githubusercontent.com/79898499/225006031-a5d8015d-4f39-4221-a6b5-9c510806eace.png)